aiven/resource_project.go

```diff
@@ -178,14 +188,15 @@ func resourceProjectUpdate(d *schema.ResourceData, m interface{}) error {
 	if err != nil {
 		return err
 	}
+	billingExtraText := d.Get("billing_extra_text").(string)
 	billingAddress := d.Get("billing_address").(string)
 	countryCode := d.Get("country_code").(string)
 	project, err := client.Projects.Update(
 		d.Get("project").(string),
 		aiven.UpdateProjectRequest{
 			BillingAddress:   &billingAddress,
 			BillingEmails:    contactEmailListForAPI(d, "billing_emails", false),
-			BillingExtraText: optionalStringPointer(d, "billing_extra_text"),
+			BillingExtraText: &billingExtraText,
```

Comment:
Why this change? Doesn't this just cause the value to be always updated instead of omitting when it's not set on Terraform side?

Reply:
`optionalStringPointer` is converting `""` empty string to `nil`, but we need to keep it as empty string pointer. Anyways updated this logic by using a new helper function.